src/helpers/tableCell.ts

```diff
@@ -0,0 +1,83 @@
+// (C) 2007-2018 GoodData Corporation
+import * as classNames from 'classnames';
+import { colors2Object, ISeparators, numberFormat } from '@gooddata/numberjs';
+
+import { styleVariables } from '../components/visualizations/styles/variables';
+
+import {
+    IMeasureTableHeader,
+    isAttributeCell,
+    ITableCellStyle,
+    ITableCellStyleAndFormattedValue,
+    MeasureCell,
+    TableCell,
+    TableHeader
+} from '../interfaces/Table';
+
+function getFormattedNumber(cellContent: MeasureCell, format: string, separators: ISeparators): string {
+    const parsedNumber: string | number = cellContent === null
+        ? ''
+        : parseFloat(cellContent);
+
+    return numberFormat(parsedNumber, format, undefined, separators);
+}
+
+export function getCellClassNames(rowIndex: number, columnIndex: number, isDrillable: boolean): string {
+    return classNames(
+        {
+            'gd-cell-drillable': isDrillable
+        },
+        `s-cell-${rowIndex}-${columnIndex}`,
+        's-table-cell'
+    );
+}
+
+export function getMeasureCellFormattedValue(
+    cellContent: MeasureCell,
+    format: string,
+    separators: ISeparators
+): string {
+    const formattedNumber = getFormattedNumber(cellContent, format, separators);
+
+    const { label } = colors2Object(formattedNumber);
+    return label === ''
```

Comment:
shouldn't this be "(empty value)"?

Reply:
but I like '-', because it doesn't need to be translated :)

Reply:
no